Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
`gclient sync` + generated solutions. At least this is (unfortunately) the official way to build v8. 

To give a better perspective how nice v8 is, on ubuntu this sync also fetch full toolchin to build v8 and don't even use system one (there is an option to use system, but it doesn't work. Officially.)